resources/dynamodb/dynamodb-backup.yaml

```diff
@@ -0,0 +1,7 @@
+apiVersion: dynamodb.services.k8s.aws/v1alpha1
```

Comment:
I would name the file `resources/dynamodb/backup.yaml` or even `resources/dynamodb/v1alpha1/backup.yaml` instead of repeating "dynamo-" in the filename.

That way, you can, potentially, read in these example files in the official Reference API docs and will have the resource name and API version as part of the filepath...

Reply:
I haven't updated the folder structure in the original repository but that's a good idea @jaypipes thanks!